en/qgc/video_streaming_wifi_broadcast.md

```diff
@@ -1,99 +1,118 @@
- - Limited diversity: Normal wifi limits you to the number of diversity streams that your wifi card offers.
 
-## What wifibroadcast makes different
-Wifibroadcast puts the wifi cards into monitor mode. This mode allows to send and receive arbitrary packets without association. Additionally, it is also possible to receive erroneous frames (where the checksum does not match). This way a true unidirectional connection is established which mimics the advantageous properties of an analog link. Those are:
+## Theory:
+
+Wifibroadcast puts the wifi cards into monitor mode. This mode allows to send and receive arbitrary packets without association.
+This way a true unidirectional connection is established which mimics the advantageous properties of an analog link. Those are:
 
  - The transmitter sends its data regardless of any associated receivers. Thus there is no risk of sudden video stall due to the loss of association
- - The receiver receives video as long as it is in range of the transmitter. If it gets slowly out of range the video quality degrades but does not stall. Even if frames are erroneous they will be displayed instead of being rejected.
+
+ - The receiver receives video as long as it is in range of the transmitter. If it gets slowly out of range the video quality degrades but does not stall.
+
  - The traditional scheme “single broadcaster – multiple receivers” works out of the box. If bystanders want to watch the video stream with their devices they just have to “switch to the right channel”
+
  - Wifibroadcast allows you to use several low cost receivers in parallel and combine their data to increase probability of correct data reception. This so-called software diversity allows you to use identical receivers to improve relieability as well as complementary receivers (think of one receiver with an omnidirectional antenna covering 360° and several directional antennas for high distance all working in parallel)
+
  - Wifibroadcast uses Forward Error Correction to archive a high reliability at low bandwidth requirements. It is able to repair lost or corrupted packets at the receiver.
 
 
+
 ## Hardware modification.
-Alpha WUS051NH is a high power card and eats too much current while TX. If you power it from USB will reset port on Odroid C1/C0.
+Alpha WUS051NH is a high power card and eats too much current while TX. If you power it from USB will reset port on most ARM boards.
 So you need to connect it to 5V BEC directly. You can do this two ways:
 
  1. Make a custom usb cable.
- 2. Cut a 5V wire on odroid PCB near usb port and wire it to BEC.
+ 2. Cut a 5V wire on PCB near usb port and wire it to BEC.
     Also I suggest to add 470uF low ESR capacitor (like ESC has) between power and ground to filter voltage spikes.
```

Comment:
Circuit diagram/snapshots - something to make it very easy for people to see exactly what you mean?